### PR TITLE
Validation fix

### PR DESCRIPTION
**Pull request by Hlamallama**

### CODE REVIEW

File: yal/assessments.py [hunk 1]
```diff
@@ -441,6 +441,15 @@ async def state_handle_assessment_reminder_response(self):
             return await self.go_to_state("state_pre_mainmenu")
 
     async def state_not_interested(self):
```
Comment (erikh360) [inline yal/assessments.py:443]: Is this state only used in the endline survey?

**Review by erikh360 — COMMENTED** [review]
Looks good, just the one question?

File: yal/assessments.py (hunk 1 above)
Comment (Hlamallama) [reply yal/assessments.py:443]: It is also used on the baseline

Comment (erikh360) [reply yal/assessments.py:443]: Oh ok, I guess its fine to update "endline_survey_started" to "" at that point too

**Review by erikh360 — APPROVED** [review]

**Pull request merged**